src/client/components/PasswordInput.tsx

```diff
@@ -10,8 +10,10 @@ import { ClientState } from '@/shared/model/ClientState';
 import { ClientStateContext } from '@/client/components/ClientState';
 
 type Props = {
+  label: string;
   error?: string;
-  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  supporting?: string;
```

Comment:
wonder if `supportText` or something along those lines might be a clearer name for this?

Reply:
I agree that this would be a clearer name however this property is set by Source originally so we're just proxying it through 🤷